packages/@aws-cdk/aws-ecs/lib/firelens-log-router.ts

```diff
@@ -109,14 +110,22 @@ export interface FirelensLogRouterDefinitionOptions extends ContainerDefinitionO
 function renderFirelensConfig(firelensConfig: FirelensConfig): CfnTaskDefinition.FirelensConfigurationProperty {
   if (!firelensConfig.options) {
     return { type: firelensConfig.type };
+  } else if (firelensConfig.options.configFileValue === undefined) {
+    // config file options must be set together and are optional
```

Comment:
can you be more specific in this comment. i think i understand -- if `configFileValue` isn't set, then `configFileType` doesn't need to be either. But it took some time to get there. Also, do you have a link to why this is the case, or anything like that?